apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -40,6 +40,46 @@
 // Always include these helper files for dotcom FSE.
 require_once __DIR__ . '/dotcom-fse/helpers.php';
 
+/**
+ * Load Core Site Editor.
+ */
+function load_core_site_editor() {
+	$options = get_option( 'gutenberg-experiments' );
+
+	// Initialize options array if needed.
+	if ( ! $options ) {
+		$options = array();
+	}
```

Comment:
```suggestion
	$options = get_option( 'gutenberg-experiments', [] );
```

What about just setting it to an empty array as the default? :)

Reply:
For some reason it was initialized for my sandbox site as `false`, so in that case I wanted to explicitly define it as an array instead.  I don't think that default option will work in that case.

Reply:
oh I see what you mean :) I think `false` is the normal default option, but if it's not that then your solution makes sense 👍

Reply:
The first time it returned as 'false', then after I tried to set it as 'false' again (i think it deleted the option) it started returning nothing.  My hunch is there that dotcom is explicitly setting it as `false` on site creation for that option.